### PR TITLE
TICtxtWrd-common: Test Improvement, Context Wording, for ecosystem common.

### PR DESCRIPTION
**Pull request by thavaahariharangit**
Test Improvement, Context Wording, for ecosystem common.

Writing a sensible context, for test.

### CODE REVIEW

File: common/spec/dependabot/git_commit_checker_spec.rb [hunk 1]
```diff
@@ -458,8 +458,8 @@
           end
         end
 
-        context "when the source returns a timeout" do
-          context "and is unknown" do
+        context "when a source returns a timeout" do
```
Comment (jurre) [inline common/spec/dependabot/git_commit_checker_spec.rb:461]: Seems like this should remain _the_

File: common/spec/dependabot/git_commit_checker_spec.rb [hunk 2]
```diff
@@ -522,7 +522,7 @@
           "/info/refs?service=git-upload-pack"
       end
 
-      context "that can be reached just fine" do
+      context "when source can be reached just fine" do
```
Comment (jurre) [inline common/spec/dependabot/git_commit_checker_spec.rb:525]: When the source

File: common/spec/dependabot/git_commit_checker_spec.rb [hunk 3]
```diff
@@ -568,7 +568,7 @@
...
```
Comment (jurre) [inline common/spec/dependabot/git_commit_checker_spec.rb:571]: When the source

File: common/spec/dependabot/metadata_finders/base/changelog_finder_spec.rb [hunk 4]
```diff
@@ -437,7 +437,7 @@
         end
       end
 
-      context "for a git dependency with multiple sources", :vcr do
+      describe "for a git dependency with multiple sources", :vcr do
```
Comment (jurre) [inline common/spec/dependabot/metadata_finders/base/changelog_finder_spec.rb:440]: It seems like this should be a context, why is it now describe?

**Review by jurre — COMMENTED** [review]
Nice work on this, I had a few questions and suggestions but other than that it looks good

**Comment by jurre** [comment]
Looks like some merge conflicts in this PR, I see a bunch of unrelated commits etc, could you clean that up please?

**Comment by thavaahariharangit** [comment]
https://github.com/dependabot/dependabot-core/pull/9856 is deployed and merged

**Pull request closed without merging**